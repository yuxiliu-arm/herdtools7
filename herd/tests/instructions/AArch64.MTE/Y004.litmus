AArch64 Y004
(* IRG with all ones, existance *)
{
int x=100;
0:X0=x;
}
 P0            ;
 MOV W2,#65535 ; (* 0xFFFF *)
 IRG X1,X0,X2  ;
exists 0:X1=x:t0

