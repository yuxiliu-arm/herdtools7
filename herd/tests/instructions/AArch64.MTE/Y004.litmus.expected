Test Y004 Allowed
States 1
0:X1=x:t0;
Ok
Witnesses
Positive: 1 Negative: 0
Condition exists (0:X1=x:t0)
Observation Y004 Always 1 0
Hash=6c900fc13a3b724a96e1ce198a96ba74

